pdc/apps/common/renderers.py

```diff
@@ -140,6 +141,11 @@ def format_docstring(self, view, method, docstring):
         macros = settings.BROWSABLE_DOCUMENT_MACROS
         if view:
             macros['FILTERS'] = get_filters(view)
+            if 'list' == method:
+                ordering_field = get_ordering_field(view, method)
+                if ordering_field:
+                    ordering_string = ORDERING_STRING + " %s ." % ordering_field
```

Comment:
Is it OK no markdown format here? For me, if the fields is read seems more obvious. I am not sure.